internal/service/loadbalancer/lb.go

```diff
@@ -58,11 +59,11 @@ func (r *lbResource) ImportState(ctx context.Context, req resource.ImportStateRe
 	resource.ImportStatePassthroughID(ctx, path.Root("load_balancer_no"), req, resp)
 }
 
-func (l *lbResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
```

Comment:
(trivial) This is not related to the throughput_type change. If you want to refactor this, it should be a separate change and PR.
IMHO, making a small change explained by the title is a good practice for creating a PR.